### DISCUSSION TITLE
Disable Windows key in osu! toggle QOL feature request

### DISCUSSION THREAD

**Discussion by i2021** [ppy/osu#21582] [Ideas]
Would be really nice to have a toggle to disable Windows key fully while game is enabled to reduce unnecessary frustrations. 

Would be best in my opinion for it to be a 3 way choice between:
- Windows key enabled fully
- Windows key disabled during gameplay
- Windows key disabled fully

I feel like it would not be as complicated to implement as the "Disable Windows key during gameplay" feature already exists and is implemented.

**Comment by peppy** [comment]
Can you describe your use case?
